### DISCUSSION TITLE
Farmer accepts plot with incorrect -p and -f options when plotting. I don't understand.

### DISCUSSION THREAD

**Discussion by funstuk** [Chia-Network/chia-blockchain#6775] [Support]
Hi!
On one machine that was doing **plotting (+remote harvester)**, I found that I had incorrectly specified the -**f and -p options** - they were the same. This was the pool key.
It was several terabyte plots. 
When I do "chia plots check" command I see the following picture: 
Pool public key: 
b4ffab8026391e3cfc09614834bf95277b1f2f19cb34e61955edb2bbf51211062f5d6eb1e74b6abe3cac7c8ff208319a
Farmer public key: 
b4ffab8026391e3cfc09614834bf95277b1f2f19cb34e61955edb2bbf51211062f5d6eb1e74b6abe3cac7c8ff208319a

BUT
The farmer accepts them and processes them successfully, I see attempts at proof. I've even tried mounting plots to the main farmer! 
Why is this happening? I was pretty sure this shouldn't work. 

When I creating plotting + remote harvester machine, I did:
I used mnemonic.txt and imported the certificates from the main farmer to operate the remote harvester. 
Could this be the reason?

Do I need to redo these plots?

**Comment by BrandtH22** [comment]
@funstuk , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - the plotter does not validate which farmer or pool keys (or contract) is used so it is expected that incorrectly values would not be identified in the plotting stage. You will need to replot any plots with invalid values to ensure you are able to claim the rewards.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
